css/pages/about.scss

```diff
@@ -1,2 +1,46 @@
 // About
+.page--about {
+	margin-left: 50px;
```

Comment:
Since the other pages don't have additional styles under ```page--event```, ```page--review``` you can get rid of the ```margin-left``` and ```margin-right``` here